.gitignore

```diff
@@ -156,6 +156,7 @@ ajax/libs/ng-pdfviewer/r0.2.1/
 ajax/libs/prettyCheckable/.2.1.1/
 ajax/libs/Hyphenator/HyphenatorBeta12/
 ajax/libs/Hyphenator/ersion_*
+ajax/libs/libphonenumber/libphonenumber-*/
```

Comment:
@PeterDaveHello If this line is added, I think this library will not automatically update. Is it need to discuss with author to change the tagged scheme?